docs/mutual-funds.md

```
@@ -366,10 +385,95 @@ jobs:
+          else
+            echo "SUPABASE_URL=${{ secrets.DEV_SUPABASE_URL }}" >> $GITHUB_ENV
+            echo "SUPABASE_KEY=${{ secrets.DEV_SERVICE_ROLE_KEY }}" >> $GITHUB_ENV
+          fi
+      - run: pnpm backfill:returns
+        env:
+          NEXT_PUBLIC_SUPABASE_URL: ${{ env.SUPABASE_URL }}
+          SUPABASE_SERVICE_ROLE_KEY: ${{ env.SUPABASE_KEY }}
+```
+
+> **Step order matters**: `pnpm/action-setup` must run *before* `setup-node` so that Node can detect the pnpm store location and enable the `cache: 'pnpm'` optimisation. Reversing the order silently disables caching.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)